### ISSUE TITLE
Is this project still active?

### ISSUE THREAD

**Issue by lovetoburnswhen** [j-bennet/wharfee#163]
Just wondering if this is being maintained

**Comment by j-bennet** [comment]
Not really. I had many plans for improving wharfee, and it's still my favorite project, but haven't had the time to actively maintain it.

If you or someone else is interested in taking over and becoming a maintainer, let me know. :)

**Comment by ps06756** [comment]
Hi @j-bennet I am interested in maintaining the project and update it.

**Comment by ps06756** [comment]
@lovetoburnswhen It's a bit late, but yes we are actively maintaining it. 
Will be pushing out a new release soon
